<!doctype html>
<html>
  <head>
    <meta charset="UTF-8">
    <title>UoNMaths Workshop 2022</title>
    <link href="css/mds2020.css" rel="stylesheet" type="text/css">
  </head>
  <body>
	  <div class="container">
      <header>
        <div class="primary_header">
          <h1 class="title">Workshop on Recent Challenges in Mathematical Epidemiology</h1>
          <h2 class="title">University of Nottingham, September 27, 2022</h2>
        </div>
        <nav class="secondary_header" id="menu">
          <ul>
            <li><a href="EpiWorkshopSep2022.html" style="text-decoration: none; color: aliceblue;">ABOUT</a></li>
            <li><a href="registration.html" style="text-decoration: none; color: aliceblue">REGISTRATION</a></li>
            <li><a href="schedule.html" style="text-decoration: none; color: aliceblue">SCHEDULE</a></li>
            <li><a href="contact.html" style="text-decoration: none; color: aliceblue">CONTACT</a></li>
          </ul>
        </nav>
      </header>
      <div class="new_article">
        <p>The workshop will focus on emerging challenges in mathematical epidemiology. While many of those challenges have been brought about by the ongoing COVID-19 pandemic, the aim and the scope of the workshop go beyond the pandemic. It is hoped to serve as an opportunity to re-evaluate the state of the art and the fundamentals of the discipline.</p>
      </div>
      <h1 id="Coordinates">Coordinates</h1>
      <li>Date: 9:30am to 4:30pm on 27 September, 2022.</li>
      <li>Location: B03, Monica Patridge Building, University of Nottingham, University Park, Nottingham NG7 2RD, United Kingdom</li>

      <div class="row">
        <h2 id="organizers">Organizers</h2>
        <div class="columns">
          <p class="crop"> <img src="images/Wasiur_BALA_2.jpg" alt="KhudaBukhsh.jpg" height="210" /> </p>
          <p><b style="margin-bottom: 1cm"><a href="https://www.wasiur.xyz/">Wasiur R. KhudaBukhsh</a></b></p>
          <p>School of Mathematical Sciences<br>The University of Nottingham</p>
        </div>
        <div class="columns">
          <p class="crop"> <img src="images/Kirsty.jpeg" alt="Kirsty.jpg" height="210" /> </p>
          <p><b style="margin-bottom: 1cm"><a href="https://www.nottingham.ac.uk/mathematics/people/kirsty.bolton">Kirsty
                Bolton</a> </b></p>
          <p>School of Mathematical Sciences<br>The University of Nottingham</p>
        </div>
        <div class="columns">
          <p class="crop"> <img src="images/Yordan.jpg" alt="Yordan.jpg" height="210" /> </p>
          <p><b style="margin-bottom: 1cm"><a href="https://www.yordanraykov.net/">Yordan Raykov</a> </b></p>
          <p>School of Mathematical Sciences<br>The University of Nottingham</p>
        </div>
      </div>

      <h1 id="confirmed-speakers">Confirmed speakers</h1>
      <details style="margin-bottom: 1cm">
        <summary><strong><a href="https://www.strath.ac.uk/staff/gomesgabrieladr/">Gabriela Gomes</a></strong>, University of
        Strathclyde, United Kingdom. </summary>
        <p>Title: <em>Frailty Variation in Population Dynamics</em></p>
        <p>Abstract: Selection acting on unmeasured individual variation is a common source of bias in the analysis of populations. It has
        been shown to affect measured rates of mortality, the survival of endangered species, the scope of neutral theories of
        biodiversity and molecular evolution, measured risks of diseases whether non-communicable or infectious, the efficacy of
        interventions such as vaccines or symbionts, and it may be an underappreciated cause for widely debated reproducibility
        issues in science.
        Forms of variation that respond to selection and impact population dynamics, termed frailty variation in demography,
        remain elusive in many disciplines. I will present some examples, including how this appears to affect the estimation of
        efficacy of vaccine.</p>
      </details>
      <details style="margin-bottom: 1cm">
        <summary><strong><a href="https://www.nottingham.ac.uk/mathematics/people/frank.ball">Frank Ball</a></strong>, University of
        Nottingham, United Kingdom.</summary>
        <p>Title: <em>Epidemics on networks with preventive rewiring</em></p>
        <p>Abstract: A stochastic SIR (susceptible $ \to $ infective $ \to $ recovered) epidemic model
        defined on a social network is analysed. The underlying social network is
        described initially by an Erdös-Rényi random graph but, during the course
        of the epidemic, susceptible individuals connected to infectious neighbours
        may rewire such connections as a preventive measure. A novel construction
        of the model is presented and used to both derive a deterministic model
        for epidemics started with a positive fraction initially infected and to prove
        convergence of the scaled stochastic model to that deterministic model as
        the population size $ n \to \infty $. The final size (i.e. total number of individuals infected) of the stochastic epidemic model is also studied, focussing on
        epidemics initiated by a single infective that take off and lead to a major
        outbreak. For part of the parameter space, the final fraction infected in the
        limit as $ n \to \infty $ is discontinuous in the infection rate $ \lambda $ at its threshold $ \lambda_c $,
        thus not converging to 0 as $ \lambda \to \lambda_c $. 
        </p>
        <br>
        <p>*Based on work done jointly with Tom Britton (Stockholm University)</p>
      </details>
      <details style="margin-bottom: 1cm">
        <summary><strong><a href="https://neyman.mbi.ohio-state.edu/">Grzegorz A. Rempała</a></strong>, The Ohio State University, United
        States of America.</summary>
        <p>Title: <em>Trackable species dynamics in reaction network models</em></p>
        <p>Abstract: In a stochastic reaction network setting we define a subset of species as ’trackable’ if we can consistently follow the
        fate of its individual molecules. We show that using the classical large volume limit results, we may approximate the
        dynamics of a single molecule of trackable species in a simple and computationally efficient way. We give examples on
        how this approach may be used to obtain various characteristics of single-molecule dynamics. In particular, we apply the
        idea to obtain the distribution of the number of infections in a single individual in the course of an epidemic and the
        activity time of a single enzyme molecule. Joint work with Daniele Cappelletti.
        </p>
      </details>
      <details style="margin-bottom: 1cm">
        <summary><strong><a href="https://www.maths.nottingham.ac.uk/plp/pmzpdo/">Phil O'Neill</a></strong>, The University of Nottingham, United
          Kingdom.</summary>
        <p>Title: <em>Likelihood approximations for stochastic epidemic models</em></p>
        <p>Abstract: Individual-level stochastic models for infectious diseases invariably describe how the disease is transmitted from one
        individual to another. Conversely, in real life we rarely observe transmission, but instead we observe symptoms of
        disease. In many settings this complicates statistical inference because the likelihood of the observed data is
        intractable. Although methods such as data-augmented MCMC can deal with this, they can struggle in large-population
        settings. We describe a way to approximate the likelihood using interactions between pairs of individuals in the
        population.</p>
      </details>
      <details style="margin-bottom: 1cm">
        <summary><strong><a href="https://warwick.ac.uk/fac/sci/maths/people/staff/dyson/">Louise Dyson</a></strong>, University of
        Warwick, United Kingdom. </summary>
        <p>Title: <em>To be announced</em></p>
        <p>Abstract: To be announced</p>
      </details>
      <details style="margin-bottom: 1cm">
        <summary><strong><a href="https://profiles.sussex.ac.uk/p211073-istvan-kiss">Istvan Z. Kiss</a></strong>, University of Sussex,
        United Kingdom.</summary>
        <p>Title: <em>On parameter identifiability in network-based epidemic models</em></p>
        <p>Abstract: Many models in mathematical epidemiology are developed with the aim to provide a framework
        for parameter estimation and then prediction. It is well-known that parameters are not
        always uniquely identifiable. In this paper we consider network-based mean-field models and explore
        the problem of parameter identifiability when observations about an epidemic are available.
        Making use of the analytical tractability of most network-based mean-field models, e.g., explicit
        analytical expressions for leading eigenvalue and final epidemic size, we set up the parameter
        identifiability problem as finding the solution or solutions of a system of coupled equations.
        More precisely, subject to observing/measuring growth rate and final epidemic size, we seek to
        identify parameter values leading to these measurements. We are particularly concerned with
        disentangling transmission rate from the network density. To do this we define strong and weak
        identifiability and we find that except for the simplest model, parameters cannot be uniquely
        determined, that is they are weakly identifiable. This means that there exists multiple solutions
        (a manifold of infinite measure) which give rise to model output that is close to the data. Identifying,
        formalising and analytically describing this problem should lead to a better appreciation
        of the complexity involved in fitting models with many parameters to data.</p>
      </details>
      <details style="margin-bottom: 1cm">
        <summary><strong><a href="https://www.maths.nottingham.ac.uk/plp/pmztk/">Theodore Kypraios</a></strong>, University of
        Nottingham, United Kingdom.</summary>
        <p>Title: <em>Bayesian Non-Parametric Inference for Stochastic Epidemic Models</em></p>
        <p>Abstract: The vast majority of epidemic models in the literature are parametric, meaning that they contain inherent assumptions
        about how transmission occurs in a population. However, such assumptions can be lacking in appropriate biological or
        epidemiological justification and in consequence lead to erroneous scientific conclusions and misleading predictions. In
        this talk I will give an overview of some recent developments in Bayesian non-parametric inference for stochastic
        epidemic models and discuss future challenges in this area.</p>
      </details>
      <details style="margin-bottom: 1cm">
        <summary><strong><a href="https://sejong.korea.ac.kr/mbshome/mbs/nslab/subview.do?id=nslab_020102000000">Boseung
            Choi</a></strong>, Korea University, Korea.</summary>
        <p>Title: <em>Hierarchical Bayesian model for the chemical reaction with time delay and its application to epidemics</em></p>
        <p>Abstract: Current estimation techniques for inferring reaction rates frequently rely on marginalization over unobserved processes
        and states. This approach can be computationally challenging in simple systems, leading to significant uncertainties and
        identifiability problems in parameter estimation. Indeed, complex dynamics of chemical reactions may have hidden or
        indirectly measured compartments. This makes it challenging to construct a model of the whole process. Model
        simplification, including time delay, is essential for identifying model estimation. We will require alternative
        approaches to efficiently uncover the interactions in complex biochemical networks with time delay. We propose a
        Bayesian inference framework to replace uninteresting or unobserved reactions with time delays. Although the resulting
        models are non-Markovian, recent results on stochastic systems with random delays allow us to construct the likelihoods
        of the model parameters. We utilized MCMC methods to estimate reaction rate constants efficiently and delay distribution
        parameters from simpler states. We illustrate the proposed approach’s advantages and pitfalls using a birth-death model
        with both synthetic and experimental data and show that we can robustly infer model parameters using relatively small
        measurements. We also applied the proposed method to complex epidemic models when all compartments of the epidemic model
        are not observed.</p>
      </details>
      <details style="margin-bottom: 1cm">
        <summary><strong><a href="https://udayton.edu/directory/artssciences/mathematics/wascher-matthew.php">Matthew
            Wascher</a></strong>, University of Dayton, United States of America.</summary>
        <p>Title: <em>To be announced</em></p>
        <p>Abstract: To be announced</p>
      </details>
      <details style="margin-bottom: 1cm">
        <summary><strong><a href="https://edmhill.github.io/">Edward Hill</a></strong>, University of Warwick, United Kingdom.</summary>
        <p>Title: <em>ncorporating behaviour into epidemiological models - challenges and questions</em></p>
        <p>Abstract: My personal view on the challenges and questions for epidemiological-behavioural modelling that have arisen in the midst
        of the COVID-19 pandemic. I will also discuss the themes that arose from a set of Isaac Newton Institute Newton Gateway
        workshops on "Modelling Behaviour to Inform Policy for Pandemics", hosted in November 2021 and February 2022.</p>
      </details>
      <details style="margin-bottom: 1cm">
        <summary><strong><a href="">Francesco Di Lauro</a></strong>, University of Oxford, United Kingdom.</summary>
        <p>Title: <em>Enhancing global preparedness during an ongoing pandemic from imperfect data</em></p>
        <p>Abstract: We show that, in the early stages of an emerging variant, integrating data from national genomic surveillance and global
        human mobility with large-scale epidemic modeling allows to quantify its pandemic potential, providing quantifiable
        indicators for pro-active policy interventions. We validate our framework on worldwide spreading variants and gain
        insights about the pandemic potential of BA.5 and BA.2.75 sub-lineages. Country-level epidemic intelligence is not
        enough to contrast the pandemic of respiratory pathogens such as SARS-CoV-2 and a scalable integrated approach, i.e.
        pandemic intelligence, is required to enhance global preparedness.</p>
      </details>
      <details style="margin-bottom: 1cm">
        <summary><strong><a href="https://www.maths.nottingham.ac.uk/personal/pmzah/">Alejandra Herrera</a></strong>, University of Nottingham, United Kingdom.</summary>
        <p>Title: <em>Uncertainty and error in SARS-CoV-2 epidemiological parameters inferred from population-level epidemic models</em></p>
        <p>Abstract: TBA.</p>
      </details>
      <details style="margin-bottom: 1cm">
        <summary><strong><a href="https://www.bcs.tu-darmstadt.de/team_sos/cuikai_sos.en.jsp">Kai Cui</a></strong>, Technische
        Universität Darmstadt, Germany.</summary>
        <p>Title: <em>Motif-based Mean-field Approximations and Mean-field Games for Epidemiology</em></p>
        <p>Abstract: It is well-known that mean-field equations facilitate otherwise computationally-expensive stochastic analysis of
        microscopic agent-based models for epidemic spread by considering the evolution of fractions of agents in certain
        states. However, existing mean-field approximations remain oblivious of higher-order interactions that are common in
        practice. In this talk, we begin by considering a novel higher-order mean-field approximation, capable of handling
        clustered sparse graphs and going beyond pairwise interactions via the use of motifs. In the second half of the talk we
        consider epidemics control problems on dense graphs in discrete-time, where (e.g. human) agents are self-interested and
        may take egoistic decisions. We will consider mean-field games for analyzing rational equilibrium behavior of agents
        during pandemics. Here, we will focus on discrete-time models and present a learning-based solution.</p>
      </details>
      <details style="margin-bottom: 1cm">
        <summary><strong><a href="">Liam Critcher</a></strong>, University of Nottingham, United Kingdom.</summary>
        <p>Title: <em>The impact of household structure on herd immunity</em></p>
        <p>Abstract: This work concerns vaccination, in particular the use of disease-induced herd immunity; the spread of infection can be
        considered as a targeted vaccine, with an aim to prevent a major outbreak. The required proportion to be targeted in
        order to do so is referred to as the disease-induced herd immunity level. Often heterogeneity in the population can
        lower the disease-induced herd immunity level. We consider disease-induced herd immunity in the households model and
        compare it to the classical herd immunity level in which individuals are vaccinated uniformly at random among the
        population.</p>
      </details>
	  </div>
  </body>
</html>
