<!doctype html>
<html>
  <head>
    <meta charset="UTF-8">
    <title>Contact information</title>
    <link href="css/mds2020.css" rel="stylesheet" type="text/css">
  </head>
  <body>
		<div class="container">
      <header>
        <div class="primary_header">
          <h1 class="title">Workshop on Recent Challenges in Mathematical Epidemiology</h1>
          <h2 class="title">University of Nottingham, September 27, 2022</h2>
        </div>
        <nav class="secondary_header" id="menu" style="text-align: center;">
          <ul>
            <li><a href="EpiWorkshopSep2022.html" style="text-decoration: none; color: aliceblue">ABOUT</a></li>
            <li><a href="registration.html" style="text-decoration: none; color: aliceblue">REGISTRATION</a></li>
            <li><a href="schedule.html" style="text-decoration: none; color: aliceblue">SCHEDULE</a></li>
            <li><a href="contact.html" style="text-decoration: none; color: aliceblue">CONTACT</a></li>
          </ul>
        </nav>
      </header>
      <p>The below timetable is tentative and subject to change.</p>
      <br>
      <details style="margin-bottom: 0.25cm">
        <summary><strong>09:15:00 &nbsp;&nbsp;&nbsp;&nbsp; Welcome</strong></summary>
      </details>
      <details style="margin-bottom: 0.25cm">
        <summary><strong>09:30:00 &nbsp;&nbsp;&nbsp;&nbsp;<a href="https://www.maths.nottingham.ac.uk/plp/pmzpdo/">Phil O'Neill</a></strong>, The University of
          Nottingham, United
          Kingdom.</summary>
        <p>Title: <em>Likelihood approximations for stochastic epidemic models</em></p>
        <p>Abstract: Individual-level stochastic models for infectious diseases invariably describe how the disease is
          transmitted from one
          individual to another. Conversely, in real life we rarely observe transmission, but instead we observe symptoms of
          disease. In many settings this complicates statistical inference because the likelihood of the observed data is
          intractable. Although methods such as data-augmented MCMC can deal with this, they can struggle in large-population
          settings. We describe a way to approximate the likelihood using interactions between pairs of individuals in the
          population.</p>
      </details>
      <details style="margin-bottom: 0.25cm">
        <summary><strong>09:50:00 &nbsp;&nbsp;&nbsp;&nbsp;<a href="https://www.strath.ac.uk/staff/gomesgabrieladr/">Gabriela Gomes</a></strong>, University of
          Strathclyde, United Kingdom. </summary>
        <p>Title: <em>Frailty Variation in Population Dynamics</em></p>
        <p>Abstract: Selection acting on unmeasured individual variation is a common source of bias in the analysis of
          populations. It has
          been shown to affect measured rates of mortality, the survival of endangered species, the scope of neutral theories
          of
          biodiversity and molecular evolution, measured risks of diseases whether non-communicable or infectious, the
          efficacy of
          interventions such as vaccines or symbionts, and it may be an underappreciated cause for widely debated
          reproducibility
          issues in science.
          Forms of variation that respond to selection and impact population dynamics, termed frailty variation in demography,
          remain elusive in many disciplines. I will present some examples, including how this appears to affect the
          estimation of
          efficacy of vaccine.</p>
      </details>
      <details style="margin-bottom: 0.25cm">
        <summary><strong>10:10:00 &nbsp;&nbsp;&nbsp;&nbsp;<a href="https://warwick.ac.uk/fac/sci/maths/people/staff/dyson/">Louise Dyson</a></strong>,
          University of
          Warwick, United Kingdom. </summary>
        <p>Title: <em>To be announced</em></p>
        <p>Abstract: To be announced</p>
      </details>
      <details style="margin-bottom: 0.25cm">
        <summary><strong>10:30:00 &nbsp;&nbsp;&nbsp;&nbsp;Coffee break</strong> </summary>
      </details>
      <details style="margin-bottom: 0.25cm">
        <summary><strong>10:45:00 &nbsp;&nbsp;&nbsp;&nbsp;<a href="https://neyman.mbi.ohio-state.edu/">Grzegorz A. Rempała</a></strong>, The Ohio State
          University, United
          States of America.</summary>
        <p>Title: <em>Trackable species dynamics in reaction network models</em></p>
        <p>Abstract: In a stochastic reaction network setting we define a subset of species as ’trackable’ if we can
          consistently follow the
          fate of its individual molecules. We show that using the classical large volume limit results, we may approximate
          the
          dynamics of a single molecule of trackable species in a simple and computationally efficient way. We give examples
          on
          how this approach may be used to obtain various characteristics of single-molecule dynamics. In particular, we apply
          the
          idea to obtain the distribution of the number of infections in a single individual in the course of an epidemic and
          the
          activity time of a single enzyme molecule. Joint work with Daniele Cappelletti.
        </p>
      </details>
      <details style="margin-bottom: 0.25cm">
        <summary><strong>11:05:00 &nbsp;&nbsp;&nbsp;&nbsp;<a href="https://profiles.sussex.ac.uk/p211073-istvan-kiss">Istvan Z. Kiss</a></strong>, University
          of Sussex,
          United Kingdom.</summary>
        <p>Title: <em>On parameter identifiability in network-based epidemic models</em></p>
        <p>Abstract: Many models in mathematical epidemiology are developed with the aim to provide a framework
          for parameter estimation and then prediction. It is well-known that parameters are not
          always uniquely identifiable. In this paper we consider network-based mean-field models and explore
          the problem of parameter identifiability when observations about an epidemic are available.
          Making use of the analytical tractability of most network-based mean-field models, e.g., explicit
          analytical expressions for leading eigenvalue and final epidemic size, we set up the parameter
          identifiability problem as finding the solution or solutions of a system of coupled equations.
          More precisely, subject to observing/measuring growth rate and final epidemic size, we seek to
          identify parameter values leading to these measurements. We are particularly concerned with
          disentangling transmission rate from the network density. To do this we define strong and weak
          identifiability and we find that except for the simplest model, parameters cannot be uniquely
          determined, that is they are weakly identifiable. This means that there exists multiple solutions
          (a manifold of infinite measure) which give rise to model output that is close to the data. Identifying,
          formalising and analytically describing this problem should lead to a better appreciation
          of the complexity involved in fitting models with many parameters to data.</p>
      </details>
      <details style="margin-bottom: 0.25cm">
        <summary><strong>11:25:00 &nbsp;&nbsp;&nbsp;&nbsp;<a href="https://sejong.korea.ac.kr/mbshome/mbs/nslab/subview.do?id=nslab_020102000000">Boseung
              Choi</a></strong>, Korea University, Korea.</summary>
        <p>Title: <em>Hierarchical Bayesian model for the chemical reaction with time delay and its application to
            epidemics</em></p>
        <p>Abstract: Current estimation techniques for inferring reaction rates frequently rely on marginalization over
          unobserved processes
          and states. This approach can be computationally challenging in simple systems, leading to significant uncertainties
          and
          identifiability problems in parameter estimation. Indeed, complex dynamics of chemical reactions may have hidden or
          indirectly measured compartments. This makes it challenging to construct a model of the whole process. Model
          simplification, including time delay, is essential for identifying model estimation. We will require alternative
          approaches to efficiently uncover the interactions in complex biochemical networks with time delay. We propose a
          Bayesian inference framework to replace uninteresting or unobserved reactions with time delays. Although the
          resulting
          models are non-Markovian, recent results on stochastic systems with random delays allow us to construct the
          likelihoods
          of the model parameters. We utilized MCMC methods to estimate reaction rate constants efficiently and delay
          distribution
          parameters from simpler states. We illustrate the proposed approach’s advantages and pitfalls using a birth-death
          model
          with both synthetic and experimental data and show that we can robustly infer model parameters using relatively
          small
          measurements. We also applied the proposed method to complex epidemic models when all compartments of the epidemic
          model
          are not observed.</p>
      </details>
      <details style="margin-bottom: 0.25cm">
        <summary><strong>11:45:00 &nbsp;&nbsp;&nbsp;&nbsp;Coffee break</strong> </summary>
      </details>
      <details style="margin-bottom: 0.25cm">
        <summary><strong>12:00:00 &nbsp;&nbsp;&nbsp;&nbsp;<a href="">Francesco Di Lauro</a></strong>, University of Oxford, United Kingdom.</summary>
        <p>Title: <em>Enhancing global preparedness during an ongoing pandemic from imperfect data</em></p>
        <p>Abstract: We show that, in the early stages of an emerging variant, integrating data from national genomic
          surveillance and global
          human mobility with large-scale epidemic modeling allows to quantify its pandemic potential, providing quantifiable
          indicators for pro-active policy interventions. We validate our framework on worldwide spreading variants and gain
          insights about the pandemic potential of BA.5 and BA.2.75 sub-lineages. Country-level epidemic intelligence is not
          enough to contrast the pandemic of respiratory pathogens such as SARS-CoV-2 and a scalable integrated approach, i.e.
          pandemic intelligence, is required to enhance global preparedness.</p>
      </details>
      <details style="margin-bottom: 0.25cm">
        <summary><strong>12:15:00 &nbsp;&nbsp;&nbsp;&nbsp;<a href="https://udayton.edu/directory/artssciences/mathematics/wascher-matthew.php">Matthew
              Wascher</a></strong>, University of Dayton, United States of America.</summary>
        <p>Title: <em>Estimating transmission and prevalence in populations under repeated testing</em></p>
        <p>Abstract: Repeated testing is a powerful tool for tracking and managing disease outbreaks and has seen widespread
          use throughout
          the SARS-CoV-2 pandemic. However, under such testing regimes, naïve estimates of disease prevalence (and thus also
          estimates of quantities derived from prevalence estimates) are biased. We develop a method called interval dynamic
          survival analysis (IDSA) that adapts the strategy of dynamic survival analysis to the repeated testing scenario
          where
          data consist of the test times and test results of all individuals in the population, allowing us to estimate
          transmission and prevalence while accounting for the effects of the testing regime. The basic approach of IDSA is to
          treat individuals' test results as censorings of their infectious periods. We additionally model test sensitivity
          that
          may depend on age of infection. We implement an MCMC estimation scheme and fit our model to synthetic data and to
          real
          data collected from repeated testing of undergraduates at The Ohio State University. This is joint work with Patrick
          Schnell, Wasiur KhudaBukhsh, Mikkel Quam, Joseph Tien, and Grzegorz Rempała.</p>
      </details>
      <details style="margin-bottom: 0.25cm">
        <summary><strong>12:30:00 &nbsp;&nbsp;&nbsp;&nbsp;<a href="https://www.bcs.tu-darmstadt.de/team_sos/cuikai_sos.en.jsp">Kai Cui</a></strong>, Technische
          Universität Darmstadt, Germany.</summary>
        <p>Title: <em>Motif-based Mean-field Approximations and Mean-field Games for Epidemiology</em></p>
        <p>Abstract: It is well-known that mean-field equations facilitate otherwise computationally-expensive stochastic
          analysis of
          microscopic agent-based models for epidemic spread by considering the evolution of fractions of agents in certain
          states. However, existing mean-field approximations remain oblivious of higher-order interactions that are common in
          practice. In this talk, we begin by considering a novel higher-order mean-field approximation, capable of handling
          clustered sparse graphs and going beyond pairwise interactions via the use of motifs. In the second half of the talk
          we
          consider epidemics control problems on dense graphs in discrete-time, where (e.g. human) agents are self-interested
          and
          may take egoistic decisions. We will consider mean-field games for analyzing rational equilibrium behavior of agents
          during pandemics. Here, we will focus on discrete-time models and present a learning-based solution.</p>
      </details>
      <details style="margin-bottom: 0.25cm">
        <summary><strong>12:45:00 &nbsp;&nbsp;&nbsp;&nbsp;<a href="">Liam Critcher</a></strong>, University of Nottingham, United Kingdom.</summary>
        <p>Title: <em>The impact of household structure on herd immunity</em></p>
        <p>Abstract: This work concerns vaccination, in particular the use of disease-induced herd immunity; the spread of
          infection can be
          considered as a targeted vaccine, with an aim to prevent a major outbreak. The required proportion to be targeted in
          order to do so is referred to as the disease-induced herd immunity level. Often heterogeneity in the population can
          lower the disease-induced herd immunity level. We consider disease-induced herd immunity in the households model and
          compare it to the classical herd immunity level in which individuals are vaccinated uniformly at random among the
          population.</p>
      </details>
      <details style="margin-bottom: 0.25cm">
        <summary><strong>13:00:00 &nbsp;&nbsp;&nbsp;&nbsp;Lunch break</strong> </summary>
      </details>
      <details style="margin-bottom: 0.25cm">
        <summary><strong>15:00:00 &nbsp;&nbsp;&nbsp;&nbsp;<a href="https://www.maths.nottingham.ac.uk/plp/pmztk/">Theodore Kypraios</a></strong>, University of
          Nottingham, United Kingdom.</summary>
        <p>Title: <em>Bayesian Non-Parametric Inference for Stochastic Epidemic Models</em></p>
        <p>Abstract: The vast majority of epidemic models in the literature are parametric, meaning that they contain inherent
          assumptions
          about how transmission occurs in a population. However, such assumptions can be lacking in appropriate biological or
          epidemiological justification and in consequence lead to erroneous scientific conclusions and misleading
          predictions. In
          this talk I will give an overview of some recent developments in Bayesian non-parametric inference for stochastic
          epidemic models and discuss future challenges in this area.</p>
      </details>
      <details style="margin-bottom: 0.25cm">
        <summary><strong>15:20:00 &nbsp;&nbsp;&nbsp;&nbsp;<a href="https://edmhill.github.io/">Edward Hill</a></strong>, University of Warwick, United Kingdom.
        </summary>
        <p>Title: <em>ncorporating behaviour into epidemiological models - challenges and questions</em></p>
        <p>Abstract: My personal view on the challenges and questions for epidemiological-behavioural modelling that have
          arisen in the midst
          of the COVID-19 pandemic. I will also discuss the themes that arose from a set of Isaac Newton Institute Newton
          Gateway
          workshops on "Modelling Behaviour to Inform Policy for Pandemics", hosted in November 2021 and February 2022.</p>
      </details>
      <details style="margin-bottom: 0.25cm">
        <summary><strong>15:40:00 &nbsp;&nbsp;&nbsp;&nbsp;<a href="https://www.maths.nottingham.ac.uk/personal/pmzah/">Alejandra Herrera</a> / <a
              href="https://www.nottingham.ac.uk/mathematics/people/kirsty.bolton">Kirsty Bolton</a> </strong>, University of
          Nottingham, United Kingdom.</summary>
        <p>Title: <em>Uncertainty and error in SARS-CoV-2 epidemiological parameters inferred from population-level epidemic
            models</em></p>
        <p>Abstract: We consider how modelling choices for the distributions representing host infectiousness and the
          infection-to-death
          time-scale, within deterministic population-level epidemic models, impact inferred epidemic characteristics. We
          introduce an SIR-type model with the infected population structured by `infected age', i.e. the number of days since
          first being infected, a formulation that enables distributions to be incorporated that are consistent with clinical
          data. We show that inference based on simpler models without infected age, which implicitly misspecify these
          distributions, leads to substantial errors in inferred quantities relevant to policy-making, such as the
          reproduction
          number and the impact of interventions. We consider uncertainty quantification via a Bayesian approach, implementing
          this for both synthetic and real data focusing on UK SARS-CoV-2 data in the period 15 Feb--14 Jul 2020, and
          emphasising
          circumstances where it is misleading to neglect uncertainty.</p>
      </details>
      <details style="margin-bottom: 0.25cm">
        <summary><strong>16:00:00 &nbsp;&nbsp;&nbsp;&nbsp;Coffee break</strong>
        </summary>
      </details>
      <details style="margin-bottom: 0.25cm">
        <summary><strong>16:15:00 &nbsp;&nbsp;&nbsp;&nbsp;Discussion</strong>
        </summary>
      </details>
      <details style="margin-bottom: 0.25cm">
        <summary><strong>17:00:00 &nbsp;&nbsp;&nbsp;&nbsp;Conclusion</strong>
        </summary>
      </details>
            
     </div>
  </body>
</html>

